pkg/registry/file/dynamicpathdetector/tests/analyze_opens_test.go

```
@@ -150,3 +713,231 @@ func areStringSlicesUnique(slice []string) bool {
+	t.Helper()
+	for _, r := range result {
+		for _, alt := range alternatives {
+			if r.Path == alt {
+				return
+			}
+		}
+	}
+	assert.Fail(t, fmt.Sprintf("result does not contain any of %v, got: %v", alternatives, pathsFromResult(result)))
+}
+
+func assertPathIsOneOf(t *testing.T, actual string, alternatives ...string) {
```

golangci-lint (2.11.4)

[error] func assertPathIsOneOf is unused

(unused)